h/db/__init__.py

```diff
@@ -104,21 +94,6 @@ def _session(request):  # pragma: no cover
     # always closed.
     @request.add_finished_callback
     def close_the_sqlalchemy_session(_request):
-        if (
-            len(session.transaction._connections)  # pylint: disable=protected-access
-            > 1
-        ):
-            # There appear to still be open DB connections belonging to this
-            # request. This shouldn't happen.
-            changes = tracker.uncommitted_changes() if tracker else []
-            if changes:
-                msg = "closing a session with uncommitted changes %s"
-                log.warning(msg, changes, extra={"stack": True, "changes": changes})
-            else:
-                log.warning(
-                    "closing an unclosed DB session (no uncommitted changes)",
-                    extra={"stack": True},
-                )
         # Close any unclosed DB connections.
         # This is done outside of the `if` statement above just in case: it's
```

Comment:
This comment refers to the `if` above that's now deleted, should update the comment.